app/assets/javascripts/shared/editor_toolbar.js

```diff
@@ -15,18 +15,28 @@ class EditorToolbar {
     if (!$target.is("textarea")) { console.log("Can't initialize a rich toolbar on anything but a textarea"); return; }
 
     this.$target = $target;
-    this.opts = { 'include': $target.data('rich-toolbar').split(' ') };
+    this.opts = {
+      'include': $target.data('rich-toolbar').split(' '),
+      'uploader': $target.data('rich-toolbar-uploader')
+    };
+
+    if (this.opts.include.includes('image') && this.opts.uploader === undefined) { console.log("You initialized a RichToolbar with the image uploader option but have not provided an existing uploader to utilize"); return; }
+
     this.affixes = this.affixesLibrary();
     this.init();
   }
 
   init() {
     this.$target.wrap('<div class="editor-field" data-behavior="editor-field"><div class="textarea-container"></div></div>');
-    this.$editorField = this.$target.parents('[data-behavior=editor-field]');
-    this.$editorField.prepend('<div class="editor-toolbar" data-behavior="editor-toolbar"></div>');
-    this.$editorToolbar = this.$editorField.find('[data-behavior=editor-toolbar]');
+    this.$editorField = this.$target.parents('[data-behavior~=editor-field]');
+    this.$editorField.prepend('<div class="editor-toolbar" data-behavior="editor-toolbar" data-edit="#'+ this.$target.data('id') + '"></div>');
```

Comment:
Found an interesting part where the first editor toolbar will have a `data-edit=#undefined"`. But it doesn't seem to break anything

![image](https://user-images.githubusercontent.com/18374402/88935797-b58bb680-d2b4-11ea-9257-4ae4b9eec503.png)

Reply:
@MattBudz do we need this?

Reply:
removed